<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCreateActivityWizard</class>
 <widget class="QWidget" name="FormCreateActivityWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_activityType">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ActivityTypeAuto</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Course ID: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_courseID">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget_MainWizard">
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab-bar { height: 0px; width: 600px; }</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_start">
      <attribute name="title">
       <string>main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1" colspan="3">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="6" column="0">
          <widget class="QComboBox" name="comboBox_mai_language">
           <item>
            <property name="text">
             <string>English</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>French</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>German</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C++</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>JAVA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FORTRAN</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Due Date</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="lineEdit_main_name"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QTextEdit" name="textEdit_main_descrip"/>
         </item>
         <item row="8" column="0">
          <widget class="QDateEdit" name="dateEdit_main_dueDate"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_essayproblemSet">
      <attribute name="title">
       <string>EssayPRob</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="6" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QTableView" name="tableView_essayprob_fileNamesTypes"/>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="pushButton_essayprob_addFiles">
           <property name="text">
            <string>ADD</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QPushButton" name="pushButton_essayprob_delfiles">
           <property name="text">
            <string>DELETE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Specifiy Submission Path:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Specify Solution Path:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Specify the Names and FileTypes:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit_essay_submissionpath"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit_essay_solutionPath"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_essayprob_browseSubPath">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_essayprob_browseSolPath">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_multipleChoice">
      <attribute name="title">
       <string>page_multipleChoice</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0" colspan="4">
          <widget class="QTableView" name="tableView_mc_AnswerKey"/>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QLineEdit" name="lineEdit_mc_outputPath"/>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Specify ouput path:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_mc_browseDir">
           <property name="text">
            <string>Browse for Path</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Answer Key:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="pushButton_mc_loadAnswerKey">
           <property name="text">
            <string>Load Answer Key (.CSV)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="4">
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_codningAssign">
      <attribute name="title">
       <string>CodingAssign</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="3" column="0" colspan="3">
          <widget class="QTableView" name="tableView_coding_codingTest"/>
         </item>
         <item row="5" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Enviroment:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QComboBox" name="comboBox_coding_enviroment">
           <item>
            <property name="text">
             <string>Visual Studio</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Eclipse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Qt Creator</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Racket</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="pushButton_codning_addTest">
           <property name="text">
            <string>ADD</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Coding Assingment Tests:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="pushButton_coding_delTest">
           <property name="text">
            <string>DELETE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_lateEarlyPenalties">
      <attribute name="title">
       <string>LateEarlyBonus</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_earlybonus_del">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTableView" name="tableView_earlybonus"/>
         </item>
         <item row="2" column="2" colspan="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_earlybonus_add">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Early Bonus: (MAX 3)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="1" column="0" colspan="4">
          <widget class="QTableView" name="tableView_latepenalty"/>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_latepenalty_add">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_latepenalry_del">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Late Penalty: (MAX 3)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_rubric">
      <attribute name="title">
       <string>Rubric</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_17">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_16">
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_rubric_delete">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTableView" name="tableView_rubric"/>
         </item>
         <item row="2" column="2" colspan="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_rubric_add">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Rubric: (OPTIONAL)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_confirmation">
      <attribute name="title">
       <string>END</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Finished :)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_confirm">
           <property name="font">
            <font>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Confirm</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Welcome: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_username">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create Activity Wizard!!</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_ButtonsNav">
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_Back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
